src/poll/ShowPoll.js

```diff
@@ -0,0 +1,76 @@
+import React, {Component} from 'react'
+import {Link} from 'react-router'
+import {find} from 'lodash'
+import {get} from '../helpers.js'
+
+import css from './ShowPoll.css'
+
+class ShowPoll extends Component{
+  state = {
+    poll: {},
+    totalVotes: 0,
+  }
+  componentWillMount() {
+    get(`poll/${this.props.params.id}/results`).then((response)=> {
+      document.title = `${response.data.title} – Results`
+      this.setState({
+        poll: Object.assign({}, response.data),
+        totalVotes: this.getVotesTotal(response.data.votes),
+      })
+    })
+
+  }
+  render() {
+    if(!this.state.poll.id) return null
```

Comment:
We could display a loading spinner here while the HTTP request is being processed.